examples/listbox/listbox-grouped.html

```diff
@@ -0,0 +1,243 @@
+    <li><a href="listbox-collapsible.html">Collapsible Dropdown Listbox Example</a>: Single-select collapsible listbox that expands when activated, similar to HTML <code>select</code> with the attribute <code>size=&quot;1&quot;</code>.</li>
+  </ul>
+  <section>
+    <h2 id="ex_label">Example</h2>
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <div id="ex">
+      <div class="listbox-area">
+        <div class="left-area">
+          <span id="ss_elem">Choose your animal sidekick</span>
+          <div id="ss_elem_list" tabindex="0" role="listbox" aria-labelledby="ss_elem">
+            <ul role="group" aria-labelledby="cat1">
+              <li role="presentation" id="cat1">Land</li>
+              <li id="ss_elem_1" role="option">Cat</li>
+              <li id="ss_elem_2" role="option">Dog</li>
+              <li id="ss_elem_3" role="option">Tiger</li>
+              <li id="ss_elem_4" role="option">Reindeer</li>
+              <li id="ss_elem_5" role="option">Raccoon</li>
+            </ul>
+            <ul role="group" aria-labelledby="cat2">
+              <li role="presentation" id="cat2">Water</li>
+              <li id="ss_elem_6" role="option">Dolphin</li>
+              <li id="ss_elem_7" role="option">Flounder</li>
+              <li id="ss_elem_8" role="option">Eel</li>
+            </ul>
+            <ul role="group" aria-labelledby="cat3">
+              <li role="presentation" id="cat3">Air</li>
+              <li id="ss_elem_9" role="option">Falcon</li>
+              <li id="ss_elem_10" role="option">Winged Horse</li>
+              <li id="ss_elem_11" role="option">Owl</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    <h4>Notes</h4>
+    <p>This listbox is scrollable; it has more options than its height can accommodate.</p>
+    <ol>
+      <li>
+        Scrolling only works as expected if the listbox is the options' <code>offsetParent</code>.
+        The example uses <code>position: relative</code> on the listbox to that effect.
+      </li>
+      <li>When an option is focused that isn't (fully) visible, the listbox's scroll position is updated:
+        <ol>
+          <li>If <kbd>Up Arrow</kbd> or <kbd>Down Arrow</kbd> is pressed, the previous or next option is scrolled into view.</li>
+          <li>If <kbd>Home</kbd> or <kbd>End</kbd> is pressed, the listbox scrolls all the way to the top or to the bottom.</li>
+          <li>If <code>focusItem</code> is called, the focused option will be scrolled to the top of the view if it was located above it or to the bottom if it was below it.</li>
+          <li>If the mouse is clicked on a partially visible option, it will be scrolled fully into view.</li>
```

Comment:
This isn’t happening for me in either latest stable Chrome or Safari on macOS 10.16.

Reply:
Pushing a fix + some tests for it in just a bit!

Reply:
Updated, thanks for the catch!